File: PartTwo/PartTwo/SocialMediaView.m
//
//  SocialMediaView.m
//  PartTwo
//
//  Created by Justine Gartner on 9/15/15.
//  Copyright © 2015 Mike Kavouras. All rights reserved.
//

#import "SocialMediaView.h"

@implementation SocialMediaView

-(IBAction)likeButtonTapped{
    
    [self.delegate socialMediaViewDidTapLikeButton:self];
    
}

-(IBAction)shareButtonTapped{
    
    [self.delegate socialMediaViewDidTapShareButton:self];
    
}

-(IBAction)commentButtonTapped{
    
    [self.delegate socialMediaViewDidTapCommentButton:self];
    
}


@end

File: Social/SocialMediaView.swift
//
//  SocialMediaView.swift
//  Social
//
//  Created by Jovanny Espinal on 9/14/15.
//  Copyright © 2015 Jovanny Espinal. All rights reserved.
//

import UIKit

class SocialMediaView: UIView {

    var delegate: SocialDelegate?
    
    @IBAction func commentButtonTapped() {
        delegate?.socialMediaViewDidTapCommentButton(self)
    }
    
    @IBAction func likeButtonTapped() {
        delegate?.socialMediaViewDidTapLikeButton(self)
    }

    @IBAction func shareButtonTapped() {
        delegate?.socialMediaViewDidTapShareButton(self)
    }
}
